addons/ofxUnitTests/src/ofxUnitTests.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * ofxUnitTestsApp.h
+ *
+ *  Created on: Jul 8, 2015
+ *      Author: arturo
+ */
+#pragma once
+
+#include "ofConstants.h"
+#include "ofLog.h"
+#include "ofBaseApp.h"
+#include "ofAppRunner.h"
+#include <string>
+
+class ofColorsLoggerChannel: public ofBaseLoggerChannel{
+	std::string CON_DEFAULT="\033[0m";
```

Comment:
do these flags work on non-unixes? if not, might be a good idea to formulate this in a cross-platform way.